packages/node/CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.8.0] - 2022-08-17
+### Fixed
+- Not parsing BigInt array entity fields correctly. (#1252)
+- Worker threads not working. (#1250)
```

Comment:
Shall we add more details (what wasn't working?) or are we happy with being this vague?